jina/parser.py

```diff
@@ -239,7 +242,11 @@ def set_pea_parser(parser=None):
     gp0.add_argument('--name', type=str,
                      help='the name of this pea, used to identify the pod and its logs.')
     gp0.add_argument('--identity', type=str, default=get_random_identity(),
-                     help='the identity of the sockets, default a random string')
+                     help='the identity of the sockets, default a random string (Important for load balancing messages')
+    gp0.add_argument('--pod-identity', type=str, default=get_random_identity(),
+                     help='the identity of the Pod the Pea belongs to')
+    gp0.add_argument('--flow-identity', type=str, default=None,
```

Comment:
sorry, i have to reject on this two new lines. This creates an inconsistent id, redundant.

simply `os.environ`?

Reply:
btw, if it eventually turns out you need to use `args` as variable passing, then hide it via
```
..., help='user don\'t have to know about this'  if _SHOW_ALL_ARGS else argparse.SUPPRESS)
```

Reply:
I am gonna change this